results/formats.md

```diff
@@ -0,0 +1,86 @@
+# Test and test report formats
```

Comment:
Remove this file, please; this is all covered in readme.md.

(Have you added anything new here in the last couple days? If so, please add to readme.md instead.)

Reply:
Yep will do in the next commit